stylus/utilities/shadows.styl

```diff
@@ -0,0 +1,4 @@
+box-shadow-12=0 4px 12px 0 rgba(0, 0, 0, .08)
+
+$raised-12
+    box-shadow box-shadow-12
```

Comment:
I think it would be best to keep the utilities folder for files that output utility classes. There is a `settings` folder that is perfect for this sort of declaration :). Same thing for the radius declaration!

Reply:
You're right, I changed it :)